src/_P043_ClkOutput.ino

```diff
@@ -53,25 +95,20 @@ boolean Plugin_043(byte function, struct EventStruct *event, String& string)
 
     case PLUGIN_WEBFORM_LOAD:
       {
-        String options[3];
-        options[0] = "";
-        options[1] = F("Off");
-        options[2] = F("On");
-
+ 
         for (byte x = 0; x < PLUGIN_043_MAX_SETTINGS; x++)
         {
         	addFormTextBox(String(F("Day,Time ")) + (x + 1), String(F("p043_clock")) + (x), timeLong2String(ExtraTaskSettings.TaskDevicePluginConfigLong[x]), 32);
-//          addHtml(F("<TR><TD>Day,Time "));
-//          addHtml(x+1);
-//          addHtml(F(":<TD><input type='text' name='plugin_043_clock"));
-//          addHtml(x);
-//          addHtml(F("' value='"));
-//          addHtml(timeLong2String(ExtraTaskSettings.TaskDevicePluginConfigLong[x]));
-//          addHtml("'>");
-
-          addHtml(" ");
-          byte choice = ExtraTaskSettings.TaskDevicePluginConfig[x];
-          addSelector(String(F("p043_state")) + (x), 3, options, NULL, NULL, choice);
+          if (CONFIG_PIN1 >= 0) {
+            String options[3];
+            options[0] = "";
```

Comment:
You don't need to initialize a newly created `String` object with `""`.
Here it is clearly for readability, but I'm not entirely sure what the compiler does with this (unneeded) code.
The web interface isn't executed very often, so no need for commenting it out as optimization, so just as a remark to help in future PRs ;)

Reply:
done, added comment